cmd/promtool/unittest.go

```diff
@@ -158,6 +172,33 @@ func resolveAndGlobFilepaths(baseDir string, utf *unitTestFile) error {
 			fmt.Fprintln(os.Stderr, "  WARNING: no file match pattern", rf)
 		}
 		globbedFiles = append(globbedFiles, m...)
+		for i, file := range globbedFiles {
+			stat, err := os.Stat(file)
+			if err == nil && stat.Mode()&(fs.ModeDevice|fs.ModeNamedPipe) != 0 {
```

Comment:
question: why those from `io/fs` instead of the ones from `os`, any difference?

Reply:
The os ones import from io/fs now. I don't really think it makes any difference which way around it is.